README.md

```diff
@@ -0,0 +1,2 @@
+# zeven
+This is Ninjadeves demo for Solskogen 2017 Electric forest. Made with our open source demo tool nin. 
```

Comment:
Auto-generated from this :3
![image](https://cloud.githubusercontent.com/assets/578029/26283725/9fd55180-3e2d-11e7-9696-4855f5be9490.png)